src/i18n/en/ui.ts

```diff
@@ -39,6 +39,13 @@ export default {
 	'since.addedIn': 'Added in:',
 	'since.new': 'New',
 	'since.beta': 'Beta',
+	// Used in `<Meta>`
+	'meta.addedIn': 'Added in:',
+	'meta.badge.new': 'New',
+	'meta.badge.beta': 'Beta',
```

Comment:
Since the `<Since />` logic was duplicated in `<Meta />`, I figured it might be better to duplicate the UI strings as well. But I can also reuse the existing ones if that's better.